nimterop/globals.nim

```diff
@@ -51,15 +51,15 @@ type
     nocache*, debug*, past*, preprocess*, pnim*, pretty*, recurse*: bool
 
     consts*, enums*, procs*, types*: HashSet[string]
-
-    code*, constStr*, currentHeader*, debugStr*, enumStr*, mode*, procStr*, typeStr*: string
-    sourceFile*: string # eg, C or C++ source or header file
+    constStr*, debugStr*, enumStr*, procStr*, typeStr*: string
+    code*, currentHeader*, mode*, pluginFile*, sourceFile*: string
```

Comment:
why remove the comment `eg, C or C++ source or header file` ? is it only for .c .cpp or also for .h/.hpp ?

Reply:
This is actually the header file passed to toast.

Reply:
i'll try to clarify w a better comment in future PR

Reply:
I think we need to improve the entire gState situation in the future. The global is out of control and there's only more going to get added.

Let's make that a separate effort once we have things more stable.

Reply:
=> we can track that her https://github.com/genotrance/nimterop/issues/68

Reply:
[RESOLVED]